cpp/include/cudf/detail/reduction.cuh

```diff
@@ -230,47 +230,45 @@ std::unique_ptr<scalar> reduce(InputIterator d_in,
  * @tparam InputIterator    the input column iterator
  * @tparam OffsetIterator   the offset column iterator
  * @tparam BinaryOp         the device binary operator used to reduce
+ * @tparam OutputIterator   the output column iterator
  * @tparam OutputType       the output type of reduction
  *
  * @param[in] d_in          the begin iterator to input
  * @param[in] d_offset      the begin iterator to offset
```

Comment:
Update to `d_offset_begin` / `d_offset_end`. Be sure to specify that the end is `N+1` items after the beginning.